tensorbay/sensor/intrinsics.py

```diff
@@ -417,6 +603,39 @@ def loads(cls: Type[_T], contents: Dict[str, Dict[str, float]]) -> _T:
             A :class:`CameraIntrinsics` instance containing information from
             the contents dict.
 
+        Examples:
+            >>> contents = {
+            ...     "cameraMatrix": {
+            ...         "fx": 1,
+            ...         "fy": 2,
+            ...         "cx": 3,
+            ...         "cy": 4,
+            ...     },
+            ...     "distortionCoefficients": {
+            ...         "p1": 1,
+            ...         "p2": 2,
+            ...         "k1": 3,
+            ...         "k2": 4
+            ...     },
+            ... }
+            >>> camera_intrinsics = CameraIntrinsics.loads(contents)
```

Comment:
Since the camera_intrinsics are no longer used in the following examples, it would be better to use the following style:
```lang=python
>>> CameraIntrinsics.loads(contents)
CameraIntrinsics(
...
```